rss_pipeline/transform_rss.py

```diff
@@ -0,0 +1,99 @@
+"""This script converts the XML to dataframes and cleans them"""
+from datetime import datetime
+import xml.etree.ElementTree as ET
+import pandas as pd
+
+
+def extract_info_from_bbc_articles(xml_file: str) -> pd.DataFrame:
+    """Extracts the title, description, article URL and publication date
+        from the XML file, saves it as a dataframe"""
+    tree = ET.parse(xml_file)
+    root = tree.getroot()
+
+    articles = []
+
+    for item in root.findall('.//item'):
+        article = {}
+
+        article['title'] = item.find('title').text
+        article['description'] = item.find('description').text
+        article['url'] = item.find('guid').text
+        article['pubdate'] = item.find('pubDate').text
+
+        articles.append(article)
+
+    return pd.DataFrame(articles)
+
+
+def extract_info_from_daily_mail_articles(xml_file: str) -> pd.DataFrame:
+    """Extracts the title, description, article URL and publication date
+        from the XML file, saves it as a dataframe"""
+    tree = ET.parse(xml_file)
+    root = tree.getroot()
+
+    articles = []
+
+    for item in root.findall('.//item'):
+        article = {}
+
+        article['title'] = item.find('title').text
+        article['description'] = item.find('description').text
+        article['url'] = item.find('guid').text
+        article['pubdate'] = item.find('pubDate').text
+        articles.append(article)
+
+    return pd.DataFrame(articles)
+
+
+def convert_pubdate_to_timestamp(pubdate: str):
+    """Converts the publication date to a timestamp
+        so it is ready for the database"""
```

Comment:
For a multiple line docstring the closing `"""` needs to be on its own line as per pep8 requirements https://peps.python.org/pep-0008/#documentation-strings